geopandas/geoseries.py

```diff
@@ -134,7 +140,18 @@ def __new__(cls, data=None, index=None, crs=None, **kwargs):
 
         self = super(GeoSeries, cls).__new__(cls)
         super(GeoSeries, self).__init__(data, index=index, name=name, **kwargs)
-        self.crs = crs
+        self._crs = None
```

Comment:
Similar question here: do we need to store the CRS on the GeoSeries level, or rather access it from / set the one on the GeometryArray ?

Reply:
For some reason I had to store it here, but I will explore more.